core-bundle/assets/controllers/choices-controller.js

```diff
@@ -1,21 +1,29 @@
 import { Controller } from '@hotwired/stimulus';
+import Choices from 'choices.js';
+import css from '!!css-loader!../styles/component/choices.pcss';
```

Comment:
Why don't we load the style sheet in the `backend.pcss` file where we load all other component style sheets?

```pcss
@import "component/color-picker.pcss";
@import "component/tabs.pcss";
@import "template-studio/layout.pcss";
@import "backend-search/layout.pcss";
```

Reply:
We only ever apply those specific styles to the `shadow-root` in that case:
https://github.com/m-vo/contao/blob/7736f81a9b4fa96dfcc95ed530a0b9cea589f904/core-bundle/assets/controllers/choices-controller.js#L16

Reply:
> We only ever apply those specific styles to the shadow-root in that case

That is probably why 👍 

> Choices was a contao-component, now we install it as a npm package (https://github.com/contao/contao/commit/88a718cefc658bb59076bfe663ce7d28e8e89f88). We therefore do not load the respective css/js files in be_main anymore.

I think that refers to something else.